[JIT] Only print original SourceRange on highlight

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29708 [JIT] Only print original SourceRange on highlight**

Differential Revision: [D18472089](https://our.internmc.facebook.com/intern/diff/D18472089)

Commit:
[JIT] Only print original SourceRange on highlight

[ghstack-poisoned]

zdevito reviewed (approved):
I think we should still print both, until we print full stack traces, but we should make the orig_source_range print first.

Commit:
Update on "[JIT] Only print original SourceRange on highlight"


Differential Revision: [D18472089](https://our.internmc.facebook.com/intern/diff/D18472089)

[ghstack-poisoned]

Commit:
Update on "[JIT] Only print original SourceRange on highlight"


Differential Revision: [D18472089](https://our.internmc.facebook.com/intern/diff/D18472089)

[ghstack-poisoned]

Commit:
Update on "[JIT] Only print original SourceRange on highlight"


Differential Revision: [D18472089](https://our.internmc.facebook.com/intern/diff/D18472089)

[ghstack-poisoned]

Commit:
Update on "[JIT] Only print original SourceRange on highlight"


Differential Revision: [D18472089](https://our.internmc.facebook.com/intern/diff/D18472089)

[ghstack-poisoned]
